python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/conftest.py

```diff
@@ -0,0 +1,260 @@
+                "connect_card": {
+                    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJkIjp7ImxvZ2luIjp0cnVlLCJ1c2VyIjoiX2FjY291bnR3b3J0aHkiLCJhY2NvdW50IjoiX21vb25iZWFtX2FjYyIsImdyb3VwIjoiX21vb25iZWFtIiwiY29ubmVjdG9yIjoiY29iYWx0X2VsZXZhdGlvbiIsIm1ldGhvZCI6IlBiZkNhcmQiLCJpZGVudGl0eSI6ZmFsc2V9LCJpYXQiOjE2Njc4MzA2MzZ9.YUMGUbzxW96xsKJLo4bTorqzx8Q19GTrUi3WFRFM8BU",
+                    "uri": "https://fivetran.com/connect-card/setup?auth=eyJ0eXAiOiJKV1QiLCJh...",
+                },
+                "pause_after_trial": False,
+                "data_delay_threshold": 0,
+                "data_delay_sensitivity": "LOW",
+                "schedule_type": "auto",
+                "local_processing_agent_id": "string",
+                "connect_card_config": {
+                    "redirect_uri": "https://your.site/path",
+                    "hide_setup_guide": True,
+                },
+                "hybrid_deployment_agent_id": "string",
+            }
+        ],
+        "nextCursor": "cursor_value",
+    },
+}
+
+SAMPLE_DESTINATION_DETAILS = {
+    "code": "Success",
+    "message": "Operation performed.",
+    "data": {
+        "id": "destination_id",
+        "service": "adls",
+        "region": "GCP_US_EAST4",
+        "networking_method": "Directly",
+        "setup_status": "CONNECTED",
+        "daylight_saving_time_enabled": True,
+        "group_id": "group_id",
+        "time_zone_offset": "+3",
+        "setup_tests": [
+            {
+                "title": "Test Title",
+                "status": "PASSED",
+                "message": "Test Passed",
+                "details": "Test Details",
+            }
+        ],
+        "local_processing_agent_id": "local_processing_agent_id",
+        "private_link_id": "private_link_id",
+        "hybrid_deployment_agent_id": "hybrid_deployment_agent_id",
+        "config": {
+            "tenant_id": "service_principal_tenant_id",
+            "auth_type": "PERSONAL_ACCESS_TOKEN | OAUTH2",
+            "storage_account_name": "adls_storage_account_name",
+            "connection_type": "Directly | PrivateLink | SshTunnel | ProxyAgent",
+            "catalog": "string",
+            "should_maintain_tables_in_databricks": True,
+            "http_path": "string",
+            "oauth2_secret": "string",
+            "snapshot_retention_period": "RETAIN_ALL_SNAPSHOTS | ONE_WEEK | TWO_WEEKS | FOUR_WEEKS | SIX_WEEKS",
+            "server_host_name": "string",
+            "client_id": "service_principal_client_id",
+            "prefix_path": "adls_container_path_prefix",
+            "container_name": "adls_container_name",
+            "port": 0,
+            "databricks_connection_type": "Directly | PrivateLink | SshTunnel | ProxyAgent",
+            "secret_value": "service_principal_secret_value",
+            "oauth2_client_id": "string",
+            "personal_access_token": "string",
+        },
+    },
+}
+
+SAMPLE_CONNECTOR_DETAILS = {
```

Comment:
comment on how these were generated? What is our plan if the schema changes, etc?

Reply:
Updated in [e4892e8](https://github.com/dagster-io/dagster/pull/25756/commits/e4892e84bdd3bc520af23b573d2e92b6f6fab320)